File: app/src/main/java/com/example/architechturestartercode/presentation/favmovies/presenter/FavPresenter.java
package com.example.architechturestartercode.presentation.favmovies.presenter;

import androidx.lifecycle.LiveData;

import com.example.architechturestartercode.data.movie.model.Movie;

import java.util.List;

public interface FavPresenter {
    LiveData<List<Movie>> getFavMovies();
    void deleteFavMovie(Movie movie);
}

File: app/src/main/java/com/example/architechturestartercode/presentation/favmovies/presenter/FavPresenter.kt
package com.example.architechturestartercode.presentation.favmovies.presenter

import androidx.lifecycle.LiveData
import com.example.architechturestartercode.data.movie.model.Movie

interface FavPresenter {
    fun getFavMovies(): LiveData<List<Movie>>
    fun deleteFavMovie(movie: Movie)
}

